eng/Signing.props

```diff
@@ -0,0 +1,33 @@
+<Project>
+  <!--
+    These are third party libraries that we use in Arcade. We need to sign them even if they
+    are already signed. However, they must be signed with a 3rd party certificate.
+  -->
+  <ItemGroup>
+    <FileSignInfo Include="Owin.dll" CertificateName="3PartySHA2" />
```

Comment:
You can probably get rid of everything except `owin` from this file. Maybe you'll need to keep newtonsoft if you're using that